lib/sles4sap/console_redirection.pm

```diff
@@ -43,33 +43,133 @@ sub handle_login_prompt {
     my $pwd = get_var('_SECRET_SUT_PASSWORD', $testapi::password);
     set_serial_term_prompt();
     # look for either password prompt or command prompt to appear
-    my $serial_response = wait_serial(qr/Password:\s*$|:~/, timeout => 20);
+    my $serial_response = wait_serial(qr/Password:\s*$|:~/, timeout => 20, quiet => 1);
 
     die 'Neither password not command prompt appeared.' unless $serial_response;
     # Handle password prompt if it appears
     if (grep /Password:\s*$/, $serial_response) {
         type_password $pwd;
         send_key 'ret';
         # wait for command prompt to be ready
-        die 'Command prompt did not appear within timeout' unless wait_serial(qr/:~|#|>/, timeout => 20);
+        die 'Command prompt did not appear within timeout' unless wait_serial(qr/:~|#|>/, timeout => 20, quiet => 1);
     }
     set_serial_term_prompt();    # set correct serial prompt
 }
 
 =head2 redirection_init
 
-    redirection_init();
+    redirection_init( [, ssh_user=>$ssh_user, target_ip=>$target_ip, ssh_tunnel_port=>$ssh_tunnel_port]);
+
+B<ssh_user>: Target login user - default value is defined by OpenQA parameter REDIRECT_TARGET_USER
+
+B<target_ip>: Target host IP - default value is defined by OpenQA parameter REDIRECT_TARGET_IP
+
+B<ssh_tunnel_port>: Local port used for reverse ssh tunnel, default: 22022
+
+Does initial setup for console redirection which includes:
+
+=over
+
+=item * sets BASE_VM_ID value with current machine id. This helps identifying the base (not redirected) VM.
```

Comment:
What is `current machine` ?

Reply:
Changed it to: 
```
=item * Takes machine id value from currently controlled VM and sets BASE_VM_ID. This is considered origin point of 
    console redirection and a point where function disconnect_target_from_serial() stops 'logging out'. 
```

Does it make more sense ?

Reply:
`currently controlled VM`  --> `remote VM`